LayoutTests/platform/wpe/TestExpectations

```diff
@@ -557,12 +554,10 @@ webkit.org/b/256457 imported/w3c/web-platform-tests/css/css-fonts/variations/var
 
 # Test failing since async overflow scrolling was activated in WPE.
 webkit.org/b/224596 css3/filters/clipping-overflow-scroll-with-pixel-moving-effect-on-parent.html [ ImageOnlyFailure ]
-webkit.org/b/224596 css3/filters/clipping-overflow-scroll-with-pixel-moving-effect-on.html [ ImageOnlyFailure ]
```

Comment:
The EWS run reported this test case as failed. I wonder if there is a small difference between running the layout tests locally and in the EWS. Have you used the webkit-container SDK, or are you using the Flatpak SDK? (most of the bots are still using the Flatpak SDK, although this will change soonish)

Maybe the image-only failures could be papered over by allowing some fuzz.